lib/auth/methods.go

```diff
@@ -566,18 +575,11 @@ func (s *Server) AuthenticateSSHUser(ctx context.Context, req AuthenticateSSHReq
 		return nil, trace.Wrap(err)
 	}
 
-	actualUser, err := s.AuthenticateUser(ctx, req.AuthenticateUserRequest)
+	user, err := s.AuthenticateUser(ctx, req.AuthenticateUserRequest)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	username = actualUser
 
-	// It's safe to extract the roles and traits directly from services.User as
-	// this endpoint is only used for local accounts.
```

Comment:
Keep this comment around?

```suggestion
	// It's safe to extract the roles and traits directly from services.User as
	// this endpoint is only used for local accounts.
```